assets/tests/components/routeLadder.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react"
+import renderer, { act } from "react-test-renderer"
+import RouteLadder from "../../src/components/routeLadder"
+import { Route } from "../../src/skate"
+import { TimepointsForRouteId } from "../../src/state"
+
+declare global {
+  interface Window {
+    /* eslint-disable typescript/no-explicit-any */
+    fetch: (uri: string) => Promise<any>
```

Comment:
Could this be `Promise<Response>` in order to avoid the `eslint-disable`? Or would that require specifying fields in the mock that we don't care about?

Reply:
It appears that you have to add a bunch of extra functions we don't care about to support that, so I think it's not worth it…